---
title: "Stephen Blaskowski – CV"
author: "Stephen Blaskowski"
date: "June 10, 2025"
---

<span class="name">Stephen Blaskowski</span>

<span class="info">Computational biologist working at the intersection of genomics, microbiology, and statistics.</span>

<span class="info">Seattle, WA | [stephen.blaskowski@gmail.com](mailto:stephen.blaskowski@gmail.com) | [github.com/blasks](https://github.com/blasks)</span>


## Education

### University of Washington | <location> Seattle, WA, USA </location> <time> 2018 – 2024 </time>

- Ph.D in Molecular Engineering and Sciences | Certificate in Data Science
- Dissertation: "Inference of In Situ Microbial Physiologies via Sparse Tensor Decomposition of Metatranscriptomes"
- Committee: E. Virginia Armbrust (Oceanography), Zaid Harchaoui (Statistics), Vaughn Iverson (Oceanography), William Stafford Noble (Genome Sciences), Mari K. H. Winkler (Civil and Environmental Engineering) 

### University of Colorado | <location> Boulder, CO, USA </location> <time> 2010 – 2014 </time>

- B.A. in Molecular, Cellular, and Developmental Biology | B.A. in Neuroscience | Certificate in Middle Eastern and Islamic Studies
- Exchange student for 9 months at the American University in Cairo



## Experience

### Postdoctoral Researcher <time> 2025 – present </time>

<location> Department of Oceanography, University of Washington, Seattle </location> | PI: Ginger Armbrust 

- Leading reasearch into expression of horizontally transferred genes in cyanobacteria and their role in anti-viral defense.
- Designing computational pipelines that use genomic and metatranscriptomic data to test various hypotheses.

### Graduate Research Associate <time> 2018 – 2024 </time>

<location> Department of Oceanography, University of Washington, Seattle </location> | PI: Ginger Armbrust

- Developed sparse tensor decomposition algorithm and associated Python package for analyzing microbiome data.
- Collected in situ marine data and specimins as member of 5-week open ocean expedition to eastern Equatorial Pacific abord the RV Thomas G. Thompson. 
- Mentored undergraduate researchers with computational and wet lab projects; some presented at Undergraduate Research Symposium.

### Research Associate III <time> 2016 – 2018 </time>

<location> Zymergen, Emergyville, California </location> | Supervisor: Shawn Szyjka

- Developed novel gene and strain engineering technologies alongside R&D team colleagues.
- Coordinated across departments to complete project reports and author publications and patents.
- Promoted twice in two years from Research Associate I to Research Associate III.

### Research Associate <time> 2014 – 2016 </time>

<location> U.S. Military HIV Research Program, Silver Spring, Maryland </location> | PI: Victoria Polonis

- Adapted manual antibody-response assay to high-throughput automated liquid handling platform.
- Sustained core laboratory operations, including maintaining cell lines, ordering supplies, and safeguarding samples.

### Undergraduate Researcher <time> 2013 – 2014 </time>

<location> Molecular, Cellular, and Developmental Biology, University of Colorado, Boulder </location> | PI: Shelley Copley

- Cloned library of _E. coli_ mutant strains used to investigate directed evlolution of a key metabolic pathway.
- Assisted in the development of a genome engineering procedure that was published in a peer-reviewed manuscript.



## Publications

### Peer-Reviewed

- **S. Blaskowski**, M. Roald, P. M. Berube, R. Braakman, E. V. Armbrust. "Simultaneous acclimation to nitrogen and iron scarcity in open ocean cyanobacteria revealed by sparse tensor decomposition of metatranscriptomes." *Science Advances* (2025) [10.1126/sciadv.adr4310](https://doi.org/10.1126/sciadv.adr4310)

- R. D. Groussman, **S. Blaskowski**, S. N. Coesel, E. V. Armbrust. "MarFERReT, an open-source, version-controlled reference library of marine microbial eukaryote functional genes." *Scientific Data* (2023). [10.1038/s41597-023-02842-4](https://doi.org/10.1038/s41597-023-02842-4)

- S. G. van Creveld, S. N. Coesel, **S. Blaskowski**, R. D. Groussman, M. J. Schatz, E. V. Armbrust. "Divergent functions of two clades of flavodoxin in diatoms mitigate oxidative stress and iron limitation." Elife (2023). [10.7554/eLife.84392](https://doi.org/10.7554/eLife.84392)

- C. R. Coates, **S. Blaskowski**, S. Szyjka, H. M. van Rossum, J. Vallandingham, K. Patel, Z. Serber, J. Dean. "Systematic investigation of CRISPR–Cas9 configurations for flexible and efficient genome editing in Corynebacterium glutamicum NRRL-B11474." *Journal of Industrial Microbiology and Biotechnology* (2019). [10.1007/s10295-018-2112-7](https://doi.org/10.1007/s10295-018-2112-7)

- J. Kim, A. M. Webb, J. P. Kershner, **S. Blaskowski**, S. D. Copley. "A versatile and highly efficient method for scarless genome editing in Escherichia coli and Salmonella enterica." *BMC Biotechnology* (2014). [10.1186/1472-6750-14-84](https://doi.org/10.1186/1472-6750-14-84)


### Patents

- **S. Blaskowski**, S. D. Cleto, C. Coates, A. Miller, S. Nademanee, M. Netwal, K. Patel, S. Szyjka, P. Weyman, S. H. Stonebloom, C. S. Maxwell. "Pooled genome editing in microbes." U.S. Patent 11,053,515, issued July 6, 2021.

- **S. Blaskowski**, S. D. Cleto, C. Coates, A. Miller, S. Nademanee, M. Netwal, K. Patel, S. Szyjka, P. Weyman, S. H. Stonebloom, C. S. Maxwell. "Iterative genome editing in microbes." U.S. Patent 11,053,506, issued July 6, 2021.

- **S. Blaskowski**, C. Coates, K. Patel, H. M. van Rossum, S. Szyjka. "Genome editing using crispr in corynebacterium." U.S. Patent Application 16/943,761, filed November 19, 2020.


### Software

- [Barnacle](https://github.com/blasks/barnacle): A Python package for sparse tensor decomposition. Role: Creator. [https://barnacle-py.readthedocs.io/](https://barnacle-py.readthedocs.io/)



## Mentorship

### Undergraduate Researchers | <location> University of Washington, Seattle </location>
- Jonah Valenti (Oceanography, Biology)  <time> 2022 – 2024 </time>
- Meena Shanmugam (Microbiology) <time> 2022 – 2023 </time>
- Dhruvi Joshi (Applied Mathematics, Bioengineering) <time> 2023 – 2024 </time>



## Conferences



## Awards



## Skills
